lib/esc.js

```diff
@@ -333,24 +313,29 @@ ESC.prototype.speed = function(speed) {
   }.bind(this), 1);
 
   return this;
-};
+}, "ESC.prototype.speed: Use `throttle(μs)` (544-2400μs) instead");
 
 
+/**
+ * throttle
+ *
+ * Throttle the ESC's speed by setting the pulse
+ *
+ * @param  {Integer} throttle pwmRange[0]...pwmRange[1] (full usec range)
+ *
+ * @return {ESC} instance
+ */
+ESC.prototype.throttle = function(pulse) {
+  this.write(this.pin, Fn.constrain(pulse, this.pwmRange[0], this.pwmRange[1]));
+  return this;
+};
```

Comment:
This is what it should've always been.

Reply:
Are you sure about 544-2400 being the ideal range for ESC's? That range comes from the Arduino servo library. I understand some ESC's will automatically adjust to fit the range they see, but I don't know what typical ESC's expect.

Reply:
> Are you sure about 544-2400

As a default? Sure. User code can and should use the appropriate range for whatever platform and hardware is in use.

Reply:
Upon further investigation, I believe that this range is _not_ appropriate for ESCs and will update this patch accordingly.